<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionSyncPage</class>
 <widget class="QWidget" name="VersionSyncPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>269</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxEnable">
     <property name="text">
      <string>Enable distributed version control</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelWithoutVersionControlSupport">
     <property name="text">
      <string>BasKet is built without version control (libgit2) support</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxControl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="buttonClearHistory">
          <property name="text">
           <string>Clear version history</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelHistorySize">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <!--<customwidgets>
  <customwidget>
   <class>KCModule</class>
   <extends>QWidget</extends>
   <header>kcmodule.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>-->
 <resources/>
 <connections/>
</ui>
